lib/platform/bitbucket-server/index.ts

```diff
@@ -549,6 +552,30 @@ export function addAssignees(iid: number, assignees: string[]): Promise<void> {
   return Promise.resolve();
 }
 
+function isInvalidReviewersResponse(err: any): string[] {
+  const errors = err?.response?.body?.errors || [];
+  return (
+    errors.length > 0 &&
+    errors.every(
+      (error) => error?.exceptionName === BITBUCKET_INVALID_REVIEWERS
+    )
+  );
+}
+
+function getInvalidReviewers(err: any): string[] {
+  const errors = err?.response?.body?.errors || [];
+  let invalidReviewers = [];
+  for (const error of errors) {
+    if (error?.exceptionName === BITBUCKET_INVALID_REVIEWERS) {
```

Comment:
In general the code is written very defensively, because errors are not documented, and I simply reverse engineered what the API returns. But here it's probably safe it will never be null.